src/controllers/krate/search.rs

```diff
@@ -208,3 +221,11 @@ pub fn search(req: &mut dyn Request) -> CargoResult<Response> {
         meta: Meta { total },
     }))
 }
+
+/// Takes a string that might represent a boolean in a URL and converts it into a boolean.
```

Comment:
Do we need to generalize this much? Can we just assume a specific value?

Reply:
No, we really don't. I dropped the function and changed the option.map to just look for the string "yes".